### DISCUSSION TITLE
How to create a desktop shortcut for a file or folder?

### DISCUSSION THREAD

**Discussion by aritorhamo** [FossifyOrg/General-Discussion#481] [Q&A] [answered]
Hello,

I've been trying to find out how to create a desktop shortcut for a file, but haven't succeeded with that, although creating desktop shortcuts is announced as a feature of Fossify File Manager. How do you create a shortcut? Is it possible that my oldish Android version (6.0) is a factor? The file manager version I use is 1.0.1. Thanks!

**Comment by Aga-C** [comment, marked as answer]
Shortcuts are available from at least Android 8.

**Comment by aritorhamo** [comment]
Thanks for the answer! Unfortunately Android 6.0 is the latest one that works on my phone. I think it would be helpful to add the information about the OS version limitation to the feature list (inside parenthesis for example), where creating shortcuts is mentioned as a feature. It would have saved quite a bit of time for me at least.

**Reply by Aga-C** [reply]
We're [dropping support](https://github.com/orgs/FossifyOrg/discussions/241) for Android older than 8, so there'll be no need for it.

**Reply by aritorhamo** [reply]
`We're dropping support for Android older than 8`

Oh, I'm sorry to hear that, although I understand the diminishing willingness to use resources to support the shrinking user base of older Android versions. I bought my phone at the end of 2016, and it still works fine, so I wouldn't like to buy a new one. Even the battery is the original. I have de-Googled the phone, though, to make it faster.
